MonacoEditorComponent/Monaco/Editor/AutoSurround.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace Monaco.Editor
+{
+    /// <summary>
+    /// Options for auto surrounding.
+    /// Defaults to always allowing auto surrounding.
+    /// </summary>
    [JsonConverter(typeof(AutoSurroundConverter))]
```

Comment:
```suggestion
    /// </summary>
    [JsonConverter(typeof(AutoSurroundConverter))]
```

Looks like a typo or something here from the editor?

Reply:
Yes, I don't know why many line breaks become `^M`. I think we can use `Find and Replace` to replace all `^M` to `\n`.

Reply:
Fixed via 99738ed67a60ec50fa6715103ea54c7907e6777f